src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,29 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method login called with parameters: login={}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.warn("Authentication failed for user {}", login);
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("User {} successfully logged in", login);
         return user;
     }
 
     private User findByLogin(String login) {
+        logger.debug("Searching for user with login {}", login);
         User user = new User(login, "1234");
-        // this user identifier should be set by DB. We will use dummy data for this example
         user.setUserId(2L);
+        logger.debug("User with login {} found, ID: {}", login, user.getUserId());
```

Comment:
This debug log provides confirmation that a user was found, including the user's ID, which is useful for debugging.